composer.json

```diff
@@ -46,7 +35,9 @@
         "behat/mink-selenium-driver": "*",
         "behat/mink-sahi-driver": "*",
         "behat/mink-selenium2-driver": "*",
-        "ezsystems/behatbundle": "@dev"
+        "ezsystems/behatbundle": "@dev",
+        "sensio/generator-bundle": "~2.3",
+        "egulias/listeners-debug-command-bundle": "~1.9"
```

Comment:
Not really sure, I guess it's up to you. This is why I'm asking for opinions :stuck_out_tongue_closed_eyes: